### DISCUSSION TITLE
The lazyvim tab shortcut cannot switch between tab pages

### DISCUSSION THREAD

**Discussion by CloudThinkSpace** [LazyVim/LazyVim#3986] [Q&A]
After lazyvim is installed, the default tab shortcut cannot be used to switch between tab pages。
<leader> <tab> + f,<leader> <tab> + l,<leader> <tab> + o,<leader> <tab> + ],<leader> <tab> + [
None of these shortcuts are valid

**Comment by iamxiaojianzheng** [comment]
The default shortcuts are these, and they work for me.

https://www.lazyvim.org/keymaps

| Key  | Description | Mode |
| ---- | ----------- | ---- |
| `<leader><tab>]` | Next Tab     | **n** |
| `<leader><tab>[` | Previous Tab | **n** |
